package.json

```diff
@@ -458,7 +458,7 @@
 		"package": "webpack --config webpack.config.preview.js --mode production && npm run compile && vsce package",
 		"deploy": "vsce publish -p",
 		"lint": "eslint 'src/**/*.ts' 'preview-src/**/*.ts' --format unix",
-		"lint:fix": "eslint --fix 'src/**/*.ts 'preview-src/**/*.ts' --format unix",
+		"lint:fix": "npm run lint -- --fix",
```

Comment:
I messed up the command in #452, there was a missing single quote after `'src/**/*.ts`.
To keep it DRY, I'm now calling `npm run lint` with the argument `--fix` (using `--` trick).